x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/deactivated_user_callout.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { DeactivatedUserCallout } from './';
+
+describe('DeactivatedUserCallout', () => {
+  it('renders with new', () => {
+    const wrapper = shallow(<DeactivatedUserCallout isNew />);
+
+    expect(wrapper).toMatchInlineSnapshot(`
+      <Fragment>
+        <EuiText
+          size="s"
+        >
+          <EuiIcon
+            color="warning"
+            type="alert"
+          />
+           
+          <strong>
+            User deactivated
+          </strong>
+        </EuiText>
+        <EuiSpacer
+          size="xs"
+        />
+        <EuiText
+          size="s"
+        >
+          This user is not currently active, and access has been temporarily revoked. Users can be re-activated via the User Management area of the Kibana console.
+        </EuiText>
+        <EuiSpacer />
+      </Fragment>
+    `);
+  });
+
+  it('renders with existing', () => {
+    const wrapper = shallow(<DeactivatedUserCallout isNew={false} />);
+
+    expect(wrapper).toMatchInlineSnapshot(`
```

Comment:
[Optional, just me getting on the ol unit testing soapbox] If all that's changing here is a conditional EuiSpacer, I'd consider making your assertion more specific instead of `toMatchInlineSnapshot`, e.g.

```js
  it('renders a spacer for existing users', () => {
    const wrapper = shallow(<DeactivatedUserCallout isNew={false} />);

    expect(wrapper.find(EuiSpacer)).toHaveLength(1);
  });
```

Snapshots do get somewhat annoying to maintain when they fail & have to be updated for the smallest of UI changes (e.g. a text size or color change), which IMO is an antipattern or at least not a great dev experience. They shouldn't feel brittle / should generally focus on the important things getting rendered